Makefile

```diff
@@ -1,88 +1,93 @@
-########################################################################
-# Development
 
-tools-install:
-	@./scripts/tools-install.sh
+########################################################################
+## Development
 
-tools-versions:
-	@./scripts/tools-versions.sh
+compress:
+	upx --brute -1 ./dist/carrier-linux-arm32
+	upx --brute -1 ./dist/carrier-linux-arm64
+	upx --brute -1 ./dist/carrier-linux-amd64
+	upx --brute -1 ./dist/carrier-windows-amd64
+	upx --brute -1 ./dist/carrier-darwin-amd64
 
-version:
-	@./scripts/version.sh
+build: tools embed_files lint build-amd64
 
-help:
-	@$(MAKE) -C cli help
+build-all: tools embed_files lint build-amd64 build-arm64 build-arm32 build-windows build-darwin
 
-build:
-	@$(MAKE) -C cli build
+build-all-small:
+	@$(MAKE) LDFLAGS+="-s -w" build-all
 
-build-all:
-	@$(MAKE) -C cli build-all
+build-arm32: lint
+	GOARCH="arm" GOOS="linux" go build -ldflags '$(LDFLAGS)' -o dist/carrier-linux-arm32
 
-build-all-small:
-	@$(MAKE) -C cli build-all-small
+build-arm64: lint
+	GOARCH="arm64" GOOS="linux" go build -ldflags '$(LDFLAGS)' -o dist/carrier-linux-arm64
 
-build-arm32:
-	@$(MAKE) -C cli build-arm32
+build-amd64: lint
+	GOARCH="amd64" GOOS="linux" go build -race -ldflags '$(LDFLAGS)' -o dist/carrier-linux-amd64
 
-build-arm64:
-	@$(MAKE) -C cli build-arm64
+build-windows: lint
+	GOARCH="amd64" GOOS="windows" go build -ldflags '$(LDFLAGS)' -o dist/carrier-windows-amd64
 
-build-amd64:
-	@$(MAKE) -C cli build-amd64
+build-darwin: lint
+	GOARCH="amd64" GOOS="darwin" go build -ldflags '$(LDFLAGS)' -o dist/carrier-darwin-amd64
 
-build-windows:
-	@$(MAKE) -C cli build-windows
+test: lint
+	ginkgo ./...
 
-build-darwin:
-	@$(MAKE) -C cli build-darwin
+generate:
+	go generate ./...
 
-compress:
-	upx --brute -1 ./cli/dist/carrier-linux-arm32
-	upx --brute -1 ./cli/dist/carrier-linux-arm64
-	upx --brute -1 ./cli/dist/carrier-linux-amd64
-	upx --brute -1 ./cli/dist/carrier-windows-amd64
-	upx --brute -1 ./cli/dist/carrier-darwin-amd64
+lint:	fmt vet tidy
 
-test:
-	@$(MAKE) -C cli test
+vet:
+	go vet ./...
 
-lint:
-	@$(MAKE) -C cli lint
+tidy:
+	go mod tidy
 
 fmt:
-	@$(MAKE) -C cli fmt
+	go fmt ./...
 
-vet:
-	@$(MAKE) -C cli vet
+.PHONY: tools
+tools:
+	go get github.com/rakyll/statik
 
-tidy:
-	@$(MAKE) -C cli tidy
+update_eirini:
+	mkdir -p embedded-files/eirini
+	wget https://github.com/cloudfoundry-incubator/eirini-release/releases/download/v2.0.0/eirini-yaml.tgz -O embedded-files/eirini/eirini-v2.0.0.tgz
 
-generate:
-	@$(MAKE) -C cli generate
+update_registry:
+	helm package ../container-registry/chart/container-registry/ -d embedded-files
```

Comment:
not `..` anymore?

Reply:
Ggk. Checking ... Will fix.

Reply:
@jimmykarily Huh. That is more difficult. The entire `container-registry` directory is gone, as part of removing the bash and charts. Do we have to restore it ? How was that directory filled ? Maybe we should put that command here, with the directory being temporary ?! ... Some kind of `helm fetch` ?

Reply:
Fixed with commit ac32764269530c7676cb85093d5c2e4d83769199.